css/src/components/dropdown.css

```diff
@@ -56,6 +56,13 @@
   box-sizing: border-box;
 }
 
+.Dropdown--errorWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
All these classes already exists in the design system. Please remove this

Reply:
Done